pkg/component/component.go

```diff
@@ -245,6 +245,12 @@ func CreateFromPath(client *occlient.Client, params occlient.CreateArgs) error {
 		return err
 	}
 
+	if params.Wait {
+		podSelector := fmt.Sprintf("deploymentconfig=%s-%s", labels["app.kubernetes.io/component-name"], labels["app"])
```

Comment:
this should use constants from labels package.
And also this function https://github.com/redhat-developer/odo/blob/151760818982a65491425422ac5ad8fb28e44ae2/pkg/util/util.go#L47:6